Sources/RouterServiceInterface/Feature.swift

```diff
@@ -1,11 +1,22 @@
 import UIKit
 
 public protocol Feature {
-    func build(
-        fromRoute route: Route?
-    ) -> UIViewController
-
+    func build(fromRoute route: Route?) -> UIViewController
     func resolve(withStore store: StoreInterface)
+    
+    /// Provides if feature is enabled to be presented, should be used when feature is controlled by toggle (feature flag, user defaults, etc)
+    /// Overwrite this method to handle the logic for presentation because
+    /// default value is always `true` (enabled)
```

Comment:
```suggestion
    /// By default, this method returns `true` (enabled). Overwrite it to manually handle the presentation logic.
    /// If your feature can be disabled, you must also implement `fallback(_:)` to provide the alternate feature to be presented.
```

English and adding info about fallbacks here